app/controllers/conferences_controller.rb

```diff
@@ -20,36 +20,40 @@ def show
     authorize! :show, @conference # TODO: reduce the 10 queries performed here
 
     splashpage = @conference.splashpage
-    if splashpage.include_cfp
-      cfps = @conference.program.cfps
-      @call_for_events = cfps.find { |call| call.cfp_type == 'events' }
-      if @call_for_events.try(:open?)
-        @event_types = @conference.event_types.pluck(:title)
-        @track_names = @conference.confirmed_tracks.pluck(:name).sort
+    if !splashpage.present?
+      redirect_to admin_conference_splashpage_path(@conference.short_title)
```

Comment:
Instead of upsetting the whole page, consider a short-circuit condition:

```
unless splashpage.present?
    redirect_to admin_conference_splashpage_path(@conference.short_title) and return
end
```

Reply:
Ok, will do. Thanks for the feedback!